### DISCUSSION TITLE
Can I add a copy to RTF command？

### DISCUSSION THREAD

**Discussion by HeliusHao** [massCodeIO/massCode#90] [Q&A]
![image](https://user-images.githubusercontent.com/33107305/173071450-50d624f4-2db9-46a4-aaf4-8243e2a3e730.png)
![image](https://user-images.githubusercontent.com/33107305/173071488-7c9158a0-2d4a-40ff-aae9-043b6440357a.png)
thiank

**Comment by HeliusHao** [comment]
Can you add a 【copy to rtf】or 【copy RTF to clipboard】 command, think

**Comment by sangyuxiaowu** [comment]
是以富文本复制到剪贴板？
跟vscode代码高亮可以复制到word类似吧

**Reply by HeliusHao** [reply]
yes
